src/Shopway.Presentation/Controllers/UsersController.cs

```diff
@@ -146,7 +146,7 @@ CancellationToken cancellationToken
 
     [HttpPost("[action]")]
     [Authorize]
-    [ProducesResponseType<Ok>(StatusCodes.Status200OK)]
```

Comment:
Do not remove this generic parameter

Reply:
@dr-marek-jaskula 
I have applied your comment.
Thank you for your consideration.